src/ApiResponse.php

```diff
@@ -135,10 +135,10 @@ public static function validateOrFail(
     }
 
     /**
-    * Add data to response and transform according to its type.
-    *
-    * @param array|Collection|JsonResource|ResourceCollection $data
```

Comment:
i think we should keep the collection just because its more popular rather than `Arrayable`

```php
@param array|Arrayable|Collection|JsonResource|ResourceCollection
```